model/model.js

```diff
@@ -13,16 +14,56 @@ exports.selectCategories = () => {
     });
 };
 
-exports.selectReviews = () => {
+exports.selectReviews = (query) => {
+  let order_by = `desc`
+  let sort_by = `created_at`;
+  if (query.sort_by) {
+    if (
+      query.sort_by !== `title` &&
+      query.sort_by !== `designer` && 
+      query.sort_by !== `owner` &&
+      query.sort_by !== `review_img_url` &&
+      query.sort_by !== `review_body` &&
+      query.sort_by !== `category` &&
+      query.sort_by !== `created_at` &&
+      query.sort_by !== `votes`
+    ) {
```

Comment:
This works fine but have a think about if there is a cleaner way to do this check with an array and an array method?